src/types/index.ts

```diff
@@ -85,8 +89,11 @@ export type ResultColumn = {
 
 export type ResultRow = Value[];
 
+export type NonNullValue = boolean | number | string;
 export type Value = boolean | number | string | null;
 
+export type TooltipFunction = (v: NonNullValue) => string;
```

Comment:
Why do you need to accept `boolean | number | string` here instead of just `number`?